tests/src/test/scala/whisk/core/cli/test/WskConfigTests.scala

```diff
@@ -149,9 +149,7 @@ class WskConfigTests
                 // Send request to https://<apihost>/api/v1/namespaces, wsk client passes client certificate to nginx, nginx will
                 // verify it by client ca's openwhisk-client-ca-cert.pem
                 val stdout = wsk.cli(Seq("property", "set", "-i", "--apihost", wskprops.apihost, "--auth", wskprops.authKey,
-                    "--cert", wskprops.cert, "--key", wskprops.key, "--namespace", namespace), env = env).stdout
```

Comment:
@drcariel not that you in fact changed the test to get it to pass. This is regressing functionality.